components/combobox/metadata/combobox.yml

```diff
@@ -0,0 +1,279 @@
+name: Combobox
+description: Comboboxes combine a text field with a picker menu.
+SpectrumSiteSlug: https://spectrum.adobe.com/page/combo-box/
+sections:
+  - name: Migration Guide
+    description: |
+      ### Component separated from InputGroup
+      This component was previously within InputGroup. It is now separated out into its own component (v1.0.0).
+      The classes containing `InputGroup` have been renamed or removed. For details, see the "renamed" and "removed" sections below or the example markup.
+
+      ### New Picker markup
+      Instead of a `.spectrum-Picker`, Combobox now uses `.spectrum-PickerButton`. Refer to the example markup for usage details.
+
+      ### New Textfield markup
+      Combobox now uses the new Textfield markup. See the [Textfield migration guide](textfield.html#migrationguide) for more information.
+
+      ### Additional clases
+      The following classes must be added:
+
+      * `.spectrum-Combobox-textfield` is now required on the Textfield outer element (`.spectrum-Textfield`)
+      * `.spectrum-Combobox-input` is now required on the `<input>` element inside of Textfields (`.spectrum-Textfield-input`)
+      * `.spectrum-Combobox-button` is now required on the FieldButton (`.spectrum-ActionButton spectrum-ActionButton--sizeM`)
+
+      ### Indicating validity and focus
+      Validity and focus must be bubbled up to the parent so descendants siblings can be styled.
+
+      Thus, implementations must add the following classes in the following situations:
+
+      * `.spectrum-Combobox.is-focused` - when the input or button is focused with the mouse
+      * `.spectrum-Combobox.is-keyboardFocused` - when the input or button is focused with the keyboard
+      * `.spectrum-Combobox.is-valid` - when the input has an explicit valid state
+      * `.spectrum-Combobox.is-invalid` - when the input has an explicit invalid state
+      * `.spectrum-Combobox.is-disabled` - when the control is disabled. You must also add `.is-disabled` to the `.spectrum-Combobox-textfield` and `[disabled]` to the `.spectrum-Combobox-button`.
+      * `.spectrum-Combobox.is-loading` - when the progress circle is being shown
+
+      ### Renamed classes
+
+      * v1.0.0: `.spectrum-InputGroup-textfield` is now `.spectrum-Combobox-textfield`, `.spectrum-InputGroup-input` is now `.spectrum-Combobox-input`, and `.spectrum-InputGroup-button` is now `.spectrum-Combobox-button`
+
+      ### Removed classes
+
+      * v1.0.0: The class `.InputGroup` is no longer used and can be be removed from the parent element
+
+examples:
+  - id: combobox
+    name: Standard
+    markup: |
+      <h4>Default</h4>
```

Comment:
Should this be wrapped in:

```
      <div class="spectrum-Examples">
        <div class="spectrum-Examples-item">
          <h4 class="spectrum-Heading spectrum-Heading--sizeXS spectrum-Examples-itemHeading">Default</h4>

          <div class="spectrum-Examples-itemGroup">...</div>
        </div>
      </div>
```

Reply:
I've updated the example markup in [eae40ab](https://github.com/adobe/spectrum-css/pull/1556/commits/eae40ab54a5667f26f7a885210234c6929a09a70) -- let me know what you think of how the docs are laid out now. I reorganized the sections a little.

Would it be okay to remove the `class="spectrum-Examples-itemGroup"` for these that only have one item? You'll see I had to add a `display:block` style for the "With Field Label" ones, because that class is making things flex for the use case of multiple items within the group (like action button). It feels a little yucky to add an inline style when it's not really needed.